src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -71,6 +71,8 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
             // shouldn't reach here
             _ => throw new ArgumentException($"Unknown browser : {Arguments.Browser}")
         };
+        // Browser respects LANGUAGE in the first place, only if empty it checks LANG
+        Environment.SetEnvironmentVariable("LANGUAGE", Arguments.Locale);
```

Comment:
This one stays?

Reply:
For this one I haven't figured out the way yet, reading into it